lib/sprockets/server.rb

```diff
@@ -23,7 +23,7 @@ def call(env)
       start_time = Time.now.to_f
       time_elapsed = lambda { ((Time.now.to_f - start_time) * 1000).to_i }
 
-      if env['REQUEST_METHOD'] != 'GET'
+      if !['GET', 'HEAD'].include?(env['REQUEST_METHOD'])
```

Comment:
@annismckenzie that will affect development only in most of the cases - that's true. but I don't see why not do this. It makes code look more readable as well. So I'm really can't understand your hostility...